packages/core/helpers/ver/4.24.7.ts

```diff
@@ -0,0 +1,19 @@
+import _ from 'lodash'
+
+const update_4_24_7 = config => {
+  const ver = '4.24.7'
+
+  const newConfig = _.cloneDeep(config)
+
+  // When switching between old version of equal number, and the revised equal number opt in, roundToPlace needs to be set.
```

Comment:
@adamdoe could you break the map change out into a function? similar to how the previous ones had been done. I have some changes for 4.24.7 also so there will be conflicts when dashboard filter stuff gets merged. if there are separate functions its a little easier to isolate.